1-3.md

```diff
@@ -21,27 +21,30 @@ navigation:
  
 Courses based on this unit:
 * Explore the rationale for organizations to address accessibility.
-* Present different testimonials from organizations that have addressed accessibility.
+* Present examples of organizations addressing accessibility.
 
 **Note:** Research relevant case studies on accessibility in organizations that are relevant to your audience. Use the resource [The Business Case for Digital Accessibility](https://www.w3.org/wai/business-case/).
 
 ## Learning Outcomes
 
 Students should be able to:
-* Identify accessibility as a driver for innovation in products.
+* List examples of technology innovations driven by accessibility.
 * Recognize accessibility as a discipline that promotes inclusion.
 * Explain how accessibility can extend market reach for organizations.
-* Identify how to minimize legal risk for some organizations.
+* Identify how accessibility can minimize legal risk for organizations.
 
 ## Competencies
 
-Skills required for this unit.
+Skills required for this unit:
 
-Students
-: Basic &mdash; Basic understanding of computers and the Web.
+Students:
+* Basic understanding of computers and the Web.
+* Basic understanding of accessibility concepts and terminology.
+* Basic understanding of how people with disabilities use the Web.
```

Comment:
I think students doing this course need some more background than just "computers and Web". Suggest we list each requirement separately for easier readability but editor's discretion.